core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -439,6 +439,10 @@ public final boolean contains(IN_NAME name) {
 
     @Override
     public final boolean contains(IN_NAME name, String value) {
+        return contains(name, value, true);
+    }
+
+    private boolean contains(IN_NAME name, String value, boolean caseSensitive) {
```

Comment:
Question) What do you think of accepting a `BiPredicate` instead to 

- Avoid double computation
- Make it more clear, since `caseSensitive` semantically doesn't make much sense for most cases

Reply:
oh~ it looks much much better. thanks for nice suggestion!

✅ updated